WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3425,10 +3425,11 @@ e_roman_empire = {
 	
 	culture = roman
 	religion = nicene
+	primary = yes
 	
 	dynasty_title_names = no
 
-	dignity = 100
+	dignity = 500
```

Comment:
It should already be primary no matter what, but to minimise bugs I want to avoid too high values. Last time we had to release a hot-fix to solve this.